source/modulo_core/test/python/translators/test_parameters.py

```diff
@@ -94,59 +94,11 @@ def test_state_parameter_read_write(state_parameters):
         assert param.get_parameter_type() == sr.ParameterType.STATE
         assert param.get_parameter_state_type() == value_types[1]
 
-        assert_np_array_equal(param.get_value().data(), value_types[0].data())
-        assert param.get_value().get_name() == value_types[0].get_name()
 
-        new_ros_param = write_parameter(param)
-        assert ros_param.name == new_ros_param.name
-        assert ros_param.type_ == new_ros_param.type_
-        assert_np_array_equal(
-            clproto.decode(clproto.from_json(new_ros_param.get_parameter_value().string_value)).data(),
-            value_types[0].data())
-
-
-def test_vector_parameter_read_write():
```

Comment:
please bring back these tests, I worked hard on them 🙏

Reply:
except that I wrote them, even though you inspired me with your C++ tests :wink:

Reply:
Ah yes, you are the Python ying to my C++ yang ☯️